thirdeye-ui/CONTRIBUTING.md

```diff
@@ -96,3 +96,27 @@ In case you end up creating a pull request with a single commit and a commit mes
 $ git commit --amend -m "<new-commit-message>"
 $ git push --force-with-lease
 ```
+
+## Creating a patch release branch
+
+When creating a patch release branch, please follow the checklist below
+
+-   Locate the appropriate [GitHub tag](https://github.com/startreedata/thirdeye/tags) corresponding to the required base version
+    -   One way to find the appropriate GitHub tag is to skim through the [**CHANGELOG**](./CHANGELOG.md)
+    -   When selecting the base version, it is important to select the most recent `PATCH` version within the required `MAJOR` or `MINOR` version (see [Semantic Versioning](https://semver.org))
+-   A patch release branch should have the following name pattern
+
+```
+<thirdeye-ui>-<base-version>.next
+```
+
+-   Patch release branch with the above name pattern will be protected with appropriate [branch protection rules](https://github.com/startreedata/thirdeye/settings/branches)
+
+-   Checkout the GitHub tag creating the patch release branch
+
+```console
+$ git checkout tags/<GitHub-tag-name> -b <patch-release-branch-name>
+```
+
+-   Update the `releaseBranch` property in [**release.config.js**](./release.config.js) on this branch with the patch release branch name
+-   Setup required build/test/release pipelines for this branch
```

Comment:
is any mention of changing the package json required here?

Reply:
Package.json needs no update. We won't be changing the tag pattern.